### ISSUE TITLE
Homebrew installation on Sierra fails

### ISSUE THREAD

**Issue by mpherg** [LoopPerfect/homebrew-lp#9]
```
» brew install loopperfect/lp/buckaroo
==> Installing buckaroo@1.3.0 from loopperfect/lp
==> Cloning https://github.com/LoopPerfect/buckaroo.git
Cloning into '/opt/boxen/cache/homebrew/buckaroo@1.3.0--git'...
remote: Counting objects: 326, done.
remote: Compressing objects: 100% (276/276), done.
remote: Total 326 (delta 97), reused 112 (delta 32), pack-reused 0
Receiving objects: 100% (326/326), 148.77 KiB | 0 bytes/s, done.
Resolving deltas: 100% (97/97), done.
Note: checking out '174d8ff0631e43c740c689082232389e1da1be20'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

==> Checking out tag v1.3.0
==> buck build :buckaroo-cli
Last 15 lines from /Users/mpherg/Library/Logs/Homebrew/buckaroo@1.3.0/01.buck:
                          ^
/private/tmp/buckaroo@1.3.0-20170814-49861-1juhlfu/src/main/java/com/loopperfect/buckaroo/tasks/CommonTasks.java:189: error: unreported exception L; must be caught or declared to be thrown
                return Either.orThrow(Serializers.parseConfig(defaulConfigString));
                                     ^
/private/tmp/buckaroo@1.3.0-20170814-49861-1juhlfu/src/main/java/com/loopperfect/buckaroo/tasks/CommonTasks.java:191: error: unreported exception L; must be caught or declared to be thrown
            return Either.orThrow(Serializers.parseConfig(EvenMoreFiles.read(configFilePath)));
                                 ^
/private/tmp/buckaroo@1.3.0-20170814-49861-1juhlfu/src/main/java/com/loopperfect/buckaroo/MoreObservables.java:-1: note: Some input files use unchecked or unsafe operations.

/private/tmp/buckaroo@1.3.0-20170814-49861-1juhlfu/src/main/java/com/loopperfect/buckaroo/MoreObservables.java:-1: note: Recompile with -Xlint:unchecked for details.

Errors: 5. Warnings: 0.

[-] BUILDING...FINISHED 16.4s [100%] (61/61 JOBS, 61 UPDATED, 61 [100.0%] CACHE MISS)
[-] DOWNLOADING... (0.00 B/S AVG, TOTAL: 0.00 B, 0 Artifacts)

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/loopperfect/homebrew-lp/issues

» java -version
java version "1.8.0_74"
Java(TM) SE Runtime Environment (build 1.8.0_74-b02)
Java HotSpot(TM) 64-Bit Server VM (build 25.74-b02, mixed mode)
```

**Comment by njlr** [comment]
I just did a clean install and did not have any issues. However, I am on a different Java version to you: 

```
$ java -version 
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)
```

We have had some issues with particular versions of Java before (see https://github.com/LoopPerfect/buckaroo/issues/68 and https://github.com/LoopPerfect/buckaroo/issues/112). 

In both cases the issue was solved by updating Java. It seems that Homebrew is shipping a broken version by default. As a quick fix, I suggest that you update Java. 

To avoid these problems in the future, I have created [an issue for providing a Packr build on macOS](https://github.com/LoopPerfect/buckaroo/issues/116). We already provide one for Linux, but I think there is value in having one for macOS too. 

Hope that helps!

**Comment by mpherg** [comment]
Thanks for the advice. I'll give that a shot and let you know.

**Comment by njlr** [comment]
Hi @mpherg 

We just released 1.3.1. Among other things, the Homebrew formula now _downloads_ rather than _builds_ the Buckaroo `.jar` file. 

To give it a try: 

```
brew update
brew install loopperfect/lp/buckaroo 
```

**Comment by mpherg** [comment]
That works great. Also updated Java cask to pull in build 1.8.0_144-b01. Thanks for the help!
